packages/frontend/src/components/ChartConfigPanel/Legend/index.tsx

```diff
@@ -7,9 +7,9 @@ import BooleanSwitch from '../../ReactHookForm/BooleanSwitch';
 import Form from '../../ReactHookForm/Form';
 import Input from '../../ReactHookForm/Input';
 import Select from '../../ReactHookForm/Select';
-import { SectionRow, SectionTitle } from './Legend.styles';
+import { InputTitle, SectionRow, SectionTitle } from './Legend.styles';
 
-const triggerSubmitFields = ['show', 'type', 'orient'];
+const triggerSubmitFields = ['show', 'orient', 'scroll'];
```

Comment:
Unsure about this change. Isn't the property still `type`? 
```methods.setValue('type', type);```

Reply:
Yes, the property on echarts is still type , but type takes a string, not a boolean. so I'm doing a conversion first from  boolean to string here https://github.com/lightdash/lightdash/pull/3770/files#diff-3e977ab56640efd9e5c882bb457bdc1f330a5e95d0239c3523219f750266359eR62

Reply:
Ok you're forcing a submit when the switch value changes 
```
                        methods.setValue('type', type);
                        handleSubmit(setLegend)();
```
But what is the `'scroll'` value in triggerSubmitFields for?

Reply:
yeah, you're right, scroll is not necessary here . I removed it